vault/logical_cubbyhole.go

```diff
@@ -43,14 +43,36 @@ type CubbyholeBackend struct {
 func (b *CubbyholeBackend) paths() []*framework.Path {
 	return []*framework.Path{
 		{
-			Pattern: ".*",
-
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   b.handleRead,
-				logical.CreateOperation: b.handleWrite,
-				logical.UpdateOperation: b.handleWrite,
-				logical.DeleteOperation: b.handleDelete,
-				logical.ListOperation:   b.handleList,
+			Pattern: framework.MatchAllRegex("path"),
+
+			Fields: map[string]*framework.FieldSchema{
+				"path": {
+					Type:        framework.TypeString,
+					Description: "Specifies the path of the secret.",
+				},
+			},
+
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
```

Comment:
One of the linters complains when you re-use unnecessary type specifiers, so you could remove these if you want.

Reply:
`framework.OperationHandler` is an interface, so these are necessary.